### PR TITLE
chore(releasing): Prepare v0.36.0 release

### PR DESCRIPTION
**Pull request by jszwedko**
Still awaiting a few changes from https://github.com/vectordotdev/vector/milestone/122. Checklist: https://github.com/vectordotdev/vector/issues/19803

### CODE REVIEW

**Review by hestonhoffman — APPROVED** [review]

File: website/cue/reference/releases/0.36.0.cue [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+			contributors: ["@esensar"]
+		},
+		{
+			type: "fix"
+			description: """
+				Fixed an issue where the `journald` source was not correctly emitting metadata when `log_namespace =
+				True`.
+
+				"""
+			contributors: ["@dalegaard"]
```
Comment (neuronull) [inline website/cue/reference/releases/0.36.0.cue:134]: Couple of "@" slipped through it seems...

Comment (jszwedko) [reply website/cue/reference/releases/0.36.0.cue:134]: Ah, yes indeed. I'll drop them manually.

Comment (jszwedko) [reply website/cue/reference/releases/0.36.0.cue]: Dropped in [0c263a9](https://github.com/vectordotdev/vector/pull/19805/commits/0c263a978fe2572653731513964975957d5c91a8)

**Review by neuronull — APPROVED** [review]

**Pull request closed without merging**